csharp-basic-topics/NullCoalescingOperator/NullCoalescingOperatorTests/NullCoalescingOperatorTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NullCoalescingOperator;
+
+namespace NullCoalescingOperatorTests
+{
+    [TestClass]
+    public class NullCoalescingOperatorTests
+    {
+        private readonly IncomeCalculator _calculator;
+        public NullCoalescingOperatorTests()
+        {
+            _calculator = new IncomeCalculator();
+        }
+
+        [DataTestMethod]
+        [DataRow(1000, 12, null, 12000)]
+        [DataRow(1000, 12, 3000, 15000)]
+        public void WhenCalculatingYearlyIncome_DefaultValueShouldBeUsedIfBonusIsNull(int monthlyIncome, int numberOfMonths, int? extraBonus, int sum)
```

Comment:
Add some newlines between Arrange/Act/Assert parts of the tests.

Reply:
Done :)